pavelib/assets.py

```diff
@@ -49,6 +49,7 @@
     'underscore.string/dist/underscore.string.js',
     'picturefill/dist/picturefill.js',
     'backbone/backbone.js',
+    'backbone.paginator/lib/backbone.paginator.js',
```

Comment:
@ehteshamkafeel I like this idea. So the UI Toolkit would make it available as, e.g.

   edx-ui-toolkit/lib/backbone.paginator.js

and that would just be made available like the UI Toolkit's own source code. I think that's a great idea.